Fix spacing around essentials

Jai-JAP opened the pull request:
Before: 

![image](https://github.com/user-attachments/assets/fb3548c4-0797-45f3-9411-1c2d0f8a6a80)

After:

![image](https://github.com/user-attachments/assets/ab8c3db4-5d9a-48e7-89ae-3a6234375887)

CosmoCreeper commented:
@Jai-JAP This should be applied when the workspace indicator is removed and I just added the spacing in a new version on this github repository.

Jai-JAP commented:
Ohh I missed the workspace indicator part. Are the spacing changes pushed? If so, case I'll just update the mod.

Jai-JAP commented:
> ... I just added the spacing in a new version on this github repository.

@CosmoCreeper I just replaced the chrome.css contents from the git head, and the spacing changes are not there.

CosmoCreeper commented:
@Jai-JAP Wdym? Could you send a picture?

CosmoCreeper commented:
@Jai-JAP Could you share your SuperPins settings? You aren't using an external tool to hide the workspace indicator are you?
